script/Arcueid/Day 9/09_00_ARC09_3_2_1.txt

```diff
@@ -629,5 +629,5 @@ I mean, without it,
 -- もともと死に<体|たい>だった遠野志貴の体は、このまま、目覚める事のない眠りに落ちていく―――
 The body of Tohno Shiki,
  which had been doomed from the start,
- falls deeper and deeper into a sleep from which it will never awaken―――
+ falls deeper and deeper into a sleep from which it will never awake―――
```

Comment:
`Awaken` feels more right to me here

Reply:
![2022-04-02-122251_2127x791_scrot](https://user-images.githubusercontent.com/2476799/161392021-1a923657-3c64-40d5-8dd9-a432de5699ae.png)

Reply:
its specifically because its _not_ "awake from" that I think this edit is good. "from which it will never awake" just feels right